<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrabbleWordFinder</class>
 <widget class="QMainWindow" name="ScrabbleWordFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrabbleWordFinder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtEnd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLetters">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>28</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>abcdefgzyxwvut</string>
    </property>
   </widget>
   <widget class="QTreeView" name="trvFound">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblEnds">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ends With:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>aw</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLetters">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>8</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Given Letters:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStarts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Starts With:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFound">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Found Words:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFind">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Words</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtContains">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>som</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLength">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblContains">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contains:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLength">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Length:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>296</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>txtLetters</tabstop>
  <tabstop>txtLength</tabstop>
  <tabstop>txtContains</tabstop>
  <tabstop>txtStart</tabstop>
  <tabstop>txtEnd</tabstop>
  <tabstop>btnFind</tabstop>
  <tabstop>trvFound</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
